docs/user/reference/metrics/string_list.md

```diff
@@ -76,13 +83,6 @@ mozilla::glean::search::engines.Add("wikipedia"_ns);
 mozilla::glean::search::engines.Add("duck duck go"_ns);
 ```
 
-**JavaScript**
-
-```js
```

Comment:
This shoud be kept as this is documenting a special API present in Firefox (FOG).